src/lib.rs

```diff
@@ -1187,7 +1190,7 @@ fn lens_iter_dates(
 ) -> Vec<DateTime<Tz>> {
     let mut lensed_dates_list: Vec<DateTime<Tz>> = Vec::new();
     for date in dates_list.iter() {
-        if date.gt(&lens_from_date) {
+        if date.ge(&lens_from_date) {
```

Comment:
For this function, the motivation was to strictly select future dates, as references in the comments.

For the functionality request, I might suggest adding a flag - perhaps, `incl_curr_date` which switches the comparison from `.gt` to `.ge`

Reply:
Yep, I think I accidentally made this change as it actually has no impact on any of the test cases. We should probably add a test case to make sure it doesn't get changed again. 😄

Reply:
agree - I'll add a test case in a seperate PR